sale_stock_on_hand_popup/views/sale_order_views.xml

```diff
@@ -1,5 +1,24 @@
 <?xml version="1.0" ?>
 <odoo>
+    <record id="view_order_form_inherit_sale_stock_qty" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field
+            name="inherit_id"
+            ref="sale_stock.view_order_form_inherit_sale_stock_qty"
+        />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='order_line']/form/group/group/div[@name='ordered_qty']/widget"
+                position="after"
+            >
+                <widget name="stock_on_hand_widget" width="0.1" />
+            </xpath>
+            <xpath expr="//field[@name='order_line']/tree/widget" position="after">
```

Comment:
**issue**: this is risky, as there can be other widgets: could you specify the name of the widget you want to find?

Reply:
Good point, fixed!